sw/device/lib/common.h

```diff
@@ -25,6 +25,8 @@ static const unsigned long UART_BAUD_RATE = 230400;
 #define FLASH_PAGES_PER_BANK 256
 #define FLASH_BANK_SZ FLASH_PAGES_PER_BANK *FLASH_PAGE_SZ
 
+#define REG_DEF(comp, id, rname) (comp##id##_BASE_ADDR + comp##_##rname##_REG)
```

Comment:
This macro wants documentation, and something like "this is for compatibility only do not use in new code".

Reply:
I agree, and will update the change if we decide to go with this PR.